.gitmodules

```diff
@@ -149,3 +149,6 @@
 [submodule "third_party/mimalloc"]
 	path = third_party/mimalloc
 	url = https://github.com/microsoft/mimalloc.git
+[submodule "third_party/sleef_fork"]
+	path = third_party/sleef_fork
+	url = https://github.com/xuhancn/sleef.git
```

Comment:
@malfet idk if you have an opinion about this, but in the past when we've needed to temporarily fork the repo, we typically updated the url in the submodule itself. For example, https://github.com/pytorch/pytorch/pull/48120

Reply:
Done, for test.